monitoring/definitions/frontend.go

```diff
@@ -62,6 +64,16 @@ func Frontend() *monitoring.Dashboard {
 				},
 				Multi: true,
 			},
+			{
+				Label: "HTTP request hostname",
+				Name:  httpRequestHostVariableName,
+				OptionsLabelValues: monitoring.ContainerVariableOptionsLabelValues{
+					Query:         "src_http_client_external_request_count",
+					LabelName:     "host",
+					ExampleOption: "ping.sourcegraph.com",
```

Comment:
I think we're phasing out ping in favor of pings, so to avoid confusion maybe?

```suggestion
					ExampleOption: "pings.sourcegraph.com",
```